app/View/ObjectTemplates/index.ctp

```diff
@@ -51,16 +51,16 @@
             <?php
                 if ($isSiteAdmin):
             ?>
-                    <th><?php echo $this->Paginator->sort('active');?></th>
+                    <th><?php echo $this->Paginator->sort('active', __('Active'));?></th>
             <?php
                 endif;
             ?>
             <th><?php echo $this->Paginator->sort('id');?></th>
             <th><?php echo $this->Paginator->sort('name');?></th>
-            <th><?php echo $this->Paginator->sort('uuid');?></th>
+            <th><?php echo $this->Paginator->sort('uuid', __('UUID'));?></th>
```

Comment:
I would refrain from translating "generic" technical terms, a UUID should always be a UUID, no matter the language, well, maybe some asian Languages might use some other names (Kanji) but they will also understand UUID

Reply:
Yes, I agree that UUID should always be a UUID.
This change I made because $this->Paginator->sort('uuid') produce column with name "Uuid" and I think that it will be better "UUID". What do you think? (Also UUID string already placed in default.pot file)

Reply:
Indeed, another option would be to simply use ('uuid', 'UUID'). Whether we want to translate it - I am not convinced either way and don't think I care enough. All decent translations would probably just translate it to UUID.